### PR TITLE
Fix build failure due to markdown and pip 18.1

### PR DESCRIPTION
**Pull request by nvzard**

### CODE REVIEW

**Comment by li-boxuan** [comment]
Is the markdown issue an upstream bug of errbot? If that's the case, we'd better raise an upstream issue ;)

**Comment by nvzard** [comment]
Yeah, I raised a query regarding that on their gitter room. Will open up an issue if I get a positive response regarding this error.

**Comment by RaiVaibhav** [comment]
I think PR is good to go :+1:

File: test-requirements.txt [hunk 1]
```diff
@@ -20,6 +20,6 @@ pytest-timeout~=1.3.0
 pytest-travis-fold~=1.3.0
 pytest-xdist~=1.15
 requests-mock~=1.2
-pip!=9.0.2, !=10.0.*, !=18.0
+pip!=9.0.2, !=10.0.*, !=18.*
```
Comment (jayvdb) [inline test-requirements.txt]: No.

We dont fix this here.

https://gitlab.com/coala/mobans/blob/master/test-requirements.txt#L23 is where the problem is.

After that is fixed, we sync it to all repos.

Comment (jayvdb) [reply test-requirements.txt]: @nvzard , it has been fixed upstream, can you update this PR accordingly please ?

**Comment by jayvdb** [comment]
ack https://github.com/coala/corobo/pull/627/commits/98d0a04a3db4923b63c167a26e6a9b7fcab446af https://github.com/coala/corobo/pull/627/commits/c4a3c7d76f3a987d4b0b242b1214c02871db28ac

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**